rts/Lua/LuaUnsyncedRead.cpp

```diff
@@ -2106,10 +2106,34 @@ namespace {
 	}
 }
 
+/***
+  * Drawing Flag
+  *
+  * @alias DrawFlag
+  * | 0 # No draw
+  * | 1 # Opaque
+  * | 2 # Alpha F
+  * | 4 # Reflection
+  * | 8 # Refraction
+  * | 16 # Sh Opaque
```

Comment:
What is "Sh"? That seems like a typo to me.

Reply:
Not typo, see me mentioning ivand to get more information.

Reply:
Yep, needs better names as you flagged above. Sorry for duplication.

Reply:
We can wait a bit and see if someone can provide a better description, if you want to merge as-is feel free to resolve all pending comments.